src/GeneralStateTestsFiller/stStaticCall/static_call_value_inheritFiller.json

```diff
@@ -0,0 +1,70 @@
+{
+    "static_call_value_inherit" : {
+        "env" : {
+            "currentCoinbase" : "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x020000",
+            "currentGasLimit" : "10000000",
+            "currentNumber" : "1",
+            "currentTimestamp" : "1000",
+            "previousHash" : "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "expect" : [
+            {
+                "indexes" : {
+                    "data" : -1,
+                    "gas" : -1,
+                    "value" : -1
+                },
+                "network" : ["Metropolis"],
+                "result" : {
+                    "094f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                        "storage" : {
+                            "0x0" : "0x1",
+                            "0x1" : "0x0"
+                        }
+                    }
+                }
+            }
+        ],
+        "pre" :
+        {
+            "a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "1000000000000",
+                "code" : "",
+                "nonce" : "0",
+                "storage" : {
+                }
+            },
+            "094f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "1",
+                "code" : "{ [[0]] (STATICCALL 50000 0xc94f5374fce5edbc8e2a8697c15331677e6ebf0b 0 0 0 32) [[1]] (MLOAD 0) }",
```

Comment:
if something goes wrong int the static call. (MLOAD would still return 0)
thats why I think using 
(if (= 0 (CALLVALUE)) (MSTORE 1 1) (SSTORE 1 2) ) 
is better. cause this way static call will return 0 if callvalue != 0  inside a staticcall